packages/theme-dark-extension/src/index.ts

```diff
@@ -20,6 +20,7 @@ const plugin: JupyterFrontEndPlugin<void> = {
     manager.register({
       name: 'JupyterLab Dark',
       isLight: false,
+      isScrollbar: true,
```

Comment:
I think a more descriptive name might be something like `themeScrollbars`

Reply:
Names are indeed hard. It actually was set to `themeScrollbars` in an earlier iteration, but then I changed it to match the other option.

I'll change it back.

Reply:
done